conf/distro/yoe.conf

```diff
@@ -84,3 +84,21 @@ ARM_INSTRUCTION_SET = "thumb"
 #PREFERRED_PROVIDER_pkgconfig = "pkgconf"
 #PREFERRED_PROVIDER_pkgconfig-native = "pkgconf-native"
 #PREFERRED_PROVIDER_nativesdk-pkgconfig = "nativesdk-pkgconf"
+
+# CONF_VERSION is increased each time build/conf/ changes incompatibly
+CONF_VERSION = "1"
+
+PREFERRED_PROVIDER_virtual/libgl ??= "mesa"
+PREFERRED_PROVIDER_virtual/psplash ??= "psplash"
+PREFERRED_PROVIDER_psplash-support ??= "psplash"
+PREFERRED_PROVIDER_jpeg ??= "libjpeg-turbo"
+PREFERRED_PROVIDER_jpeg-native ??= "libjpeg-turbo-native"
+PREFERRED_PROVIDER_grub-efi-native ??= "grub-efi-native"
+PREFERRED_PROVIDER_u-boot-fw-utils ??= "u-boot-fw-utils"
+
+# Don't generate the mirror tarball for SCM repos, the snapshot is enough
+BB_GENERATE_MIRROR_TARBALLS = "0"
```

Comment:
0 is default unless we want to set it to 1 we can drop it too. May be having  tars of git repos in dldir is good